src/pubsub/ua_pubsub_manager.h

```diff
@@ -28,11 +34,17 @@ typedef struct UA_PubSubManager {
     size_t subscribedDataSetsSize;
     TAILQ_HEAD(UA_ListOfStandaloneSubscribedDataSet, UA_StandaloneSubscribedDataSet) subscribedDataSets;
 
+    size_t topicAssignSize;
+    TAILQ_HEAD(UA_ListOfTopicAssign, UA_TopicAssign) topicAssign;
+
 #ifndef UA_ENABLE_PUBSUB_INFORMATIONMODEL
     UA_UInt32 uniqueIdCount;
 #endif
 } UA_PubSubManager;
 
+UA_StatusCode
+UA_Server_addPubSubTopicAssign(UA_Server *server, UA_ReaderGroup *readerGroup, UA_String topic);
+
```

Comment:
Maybe UA_PubSubManager_ ... would be more precise.

Reply:
Another question: Is the assign done by the user or implied by the PubSub-config? -> Why is this function needed?